### PR TITLE
TCIOS-808761 Language customisation support in SDK

### PR DESCRIPTION
**Pull request by sreedeepkesav**

### CODE REVIEW

File: TrueSDK/External/TCTrueSDK.h [hunk 1]
```diff
@@ -57,6 +57,7 @@
 
 @property (nonatomic, weak, nullable) id<TCTrueSDKDelegate> delegate;
 @property (nonatomic) TitleType titleType;
+@property (nonatomic, weak, nullable) NSString* locale;
```
Comment (vascome) [inline TrueSDK/External/TCTrueSDK.h]: I think it makes no sense to have it weak.

Comment (sreedeepkesav) [reply TrueSDK/External/TCTrueSDK.h]: Yup. Fixed

File: TrueSDK/External/TCTrueSDK.m [hunk 2]
```diff
@@ -42,16 +42,15 @@ - (void)setupWithAppKey:(nonnull NSString *)appKey
     self.appKey = appKey;
     self.appLink = appLink;
     self.titleType = TitleTypeDefault;
+    self.locale = @"en_US";
```
Comment (vascome) [inline TrueSDK/External/TCTrueSDK.m]: should not we update API to set locale explicitly?

Comment (sreedeepkesav) [reply TrueSDK/External/TCTrueSDK.m]: I don't think so. There will be a setter setLocale. This is optional. Default will always be english

Comment (vascome) [reply TrueSDK/External/TCTrueSDK.m]: OKay, then

**Pull request merged**